menu.json

```diff
@@ -0,0 +1,11 @@
+[
+    "CREATE AN ATOM"
+  , "MORE ATOMS"
+  , "MAKE IT MOLECULAR"
+  , "ORGANISMS"
+  , "TEMPLATES"
+  , "PAGES"
+  , "JSON"
+  , "SIDECAR JSON"
+  , "LET'S GET SASSY"
+]
```

Comment:
@tjheffner can you go through and fix all the new line at end of file style warnings.